apps/site/pages/en/download/current.mdx

```diff
@@ -27,7 +27,8 @@ Read the <Release.BlogPostLink>blog post</Release.BlogPostLink> for this version
 
 Learn how to <LinkWithArrow href="https://github.com/nodejs/node#verifying-binaries">verify</LinkWithArrow> signed SHASUMS.
 
-Learn how to <LinkWithArrow href="https://github.com/nodejs/node/blob/main/BUILDING.md#building-nodejs-on-supported-platforms">build Node.js</LinkWithArrow> from source.
+Download a signed <Release.DownloadLink kind="source">Node.js source</Release.DownloadLink> tarball
+and Learn how to <LinkWithArrow href="https://github.com/nodejs/node/blob/main/BUILDING.md#building-nodejs-on-supported-platforms">build Node.js</LinkWithArrow> from source.
```

Comment:
Is it useful to keep this link? `BUILDING.md` is already included in the source tarball

Reply:
Either as a link or referencing that build instructions are inside the tarball. But I believe it'd make sense to mention'em.

To be honest, I am not sure how fast people would notice that file; No strong opinions here, Im also a power user, but my thought goes to "if I am a newcomer and unzip this tarball, what would be the first document Id look for to build this source? I would glance over BUILDING.md and open it"

But this is me as a power user putting myself on the perspective of a newcomer, having that said, not sure how much that storyline actually would match reality.

Hence Im fine removing it or keeping it.

Reply:
I think it's fair to assume folks who build from source are power users.

Reply:
That's fair